data_steward/tools/combine_ehr_rdr.py

```diff
@@ -170,6 +172,88 @@ def copy_rdr_table(table):
+    max_rdr_obs_id = int(bq_utils.query(q_max_rdr_obs_id)['rows'][0]['f'][0]['v'])
+    q = ''' --Race
+            SELECT
+                ROW_NUMBER() OVER() + {offset} AS observation_id,
+                person_id,
+                observation_concept_id,
+                EXTRACT(DATE FROM observation_datetime) as observation_date,
+                observation_type_concept_id,
+                observation_datetime,
+                CAST(NULL AS FLOAT64) as value_as_number,
+                value_as_concept_id,
+                CAST(value_as_string AS STRING) as value_as_string,
+                observation_source_value,
+                observation_source_concept_id,
+                NULL as qualifier_concept_id,
+                NULL as unit_concept_id,
+                NULL as provider_id,
+                NULL as visit_occurrence_id,
+                CAST(NULL AS STRING) as unit_source_value,
+                CAST(NULL AS STRING) as qualifier_source_value,
+                NULL as value_source_concept_id,
+                CAST(NULL AS STRING) as value_source_value,
+                NULL as questionnaire_response_id
+            FROM
+            (
+              SELECT person_id, 4013886 as observation_concept_id, 38000280 as observation_type_concept_id, 
+              birth_datetime as observation_datetime,
+              race_concept_id as value_as_concept_id,
+              NULL as value_as_string,
+              race_source_value as observation_source_value, 
+              race_source_concept_id as observation_source_concept_id
+              FROM {ehr_dataset_id}.person
+
+              UNION ALL
+
+              --Ethnicity
+              SELECT person_id, 4271761 as observation_concept_id, 38000280 as observation_type_concept_id, 
+              birth_datetime as observation_datetime,
+              ethnicity_concept_id as value_as_concept_id,
+              NULL as value_as_string,
+              ethnicity_source_value as observation_source_value, 
+              ethnicity_source_concept_id as observation_source_concept_id
+              FROM {ehr_dataset_id}.person
+
+              UNION ALL
+
+              --Gender
+              SELECT person_id, 4135376 as observation_concept_id, 38000280 as observation_type_concept_id, 
+              birth_datetime as observation_datetime,
+              gender_concept_id as value_as_concept_id,
+              NULL as value_as_string,
+              gender_source_value as observation_source_value, 
+              gender_source_concept_id as observation_source_concept_id
+              FROM {ehr_dataset_id}.person
+
+              UNION ALL
+
+              --DOB
+              SELECT person_id, 4083587 as observation_concept_id, 38000280 as observation_type_concept_id, 
+              birth_datetime as observation_datetime,
+              NULL as value_as_concept_id,
+              birth_datetime as value_as_string,
+              '' as observation_source_value,
```

Comment:
@cukarthik is there a reason to use empty string instead of `NULL`?

Reply:
I thought it created an error for me when using NULL, but I can't reproduce it. so no. I'll make it NULL.